package.json

```diff
@@ -1,71 +1,31 @@
 {
-  "name": "groqd",
-  "version": "0.13.0",
-  "license": "MIT",
-  "author": {
-    "name": "Formidable",
-    "url": "https://formidable.com"
+  "name": "groqd-workspace",
+  "private": "true",
+  "workspaces": {
+    "packages": [
+      "packages/*"
+    ]
   },
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/formidablelabs/groqd.git"
-  },
-  "homepage": "https://github.com/formidablelabs/groqd",
-  "keywords": [
-    "sanity",
-    "groq",
-    "query",
-    "typescript"
-  ],
-  "main": "dist/index.js",
-  "module": "dist/index.mjs",
-  "types": "dist/index.d.ts",
-  "exports": {
-    ".": [
-      {
-        "import": "./dist/index.mjs",
-        "types": "./dist/index.d.ts",
-        "default": "./dist/index.js"
-      },
-      "./dist/index.js"
-    ],
-    "./package.json": "./package.json"
-  },
-  "files": [
-    "dist"
-  ],
   "scripts": {
-    "test:watch": "vitest",
-    "test": "vitest run",
+    "lint": "eslint packages/**/src --quiet",
+    "test": "yarn workspaces run test",
+    "test:watch": "yarn workspaces run test:watch",
     "typecheck": "tsc --noEmit",
-    "lint": "eslint src --quiet",
     "check:ci": "yarn typecheck && yarn lint && yarn test",
-    "build": "tsup",
-    "preversion": "yarn check:ci",
+    "build": "yarn workspaces run build",
     "prepare": "yarn build",
     "changeset": "changeset",
-    "version": "yarn changeset version && yarn install --frozen-lockfile"
+    "version": "yarn changeset version && yarn install"
```

Comment:
Also maybe worth adding command forwarding for the docs website at the top level package.

```
"start:docs": "yarn --cwd website start",
"build:docs": "yarn --cwd website build",
```

I don't think the docs website should be part of the monorepo setup since it isn't a shippable package, but no strong feelings either way on this.